python/lsst/ts/salkafka/component_producer_set.py

```diff
@@ -209,8 +209,14 @@ async def amain(cls):
             parser.error(f"--partitions={args.partitions} must be positive")
         if args.wait_for_ack != "all":
             args.wait_for_ack = int(args.wait_for_ack)
+        if args.username and args.password is None:
+            parser.error("You must specify --password.")
+        if args.password and args.username is None:
+            parser.error("You must specify --username.")
```

Comment:
Thank you for this nice addition. I gently suggest expanding the text a bit, e.g.:
```
"You must specify --password if you specify --username"
```
and vice-versa for the second one.